contracts/libraries/Oracle.sol

```diff
@@ -38,7 +38,8 @@ library Oracle {
             Observation({
                 blockTimestamp: blockTimestamp,
                 tickCumulative: last.tickCumulative + int56(tick) * delta,
-                liquidityCumulative: last.liquidityCumulative + uint160(liquidity) * delta,
+                secondsPerLiquidityCumulativeX128: last.secondsPerLiquidityCumulativeX128 +
```

Comment:
could we in theory just not increment in the case of liquidity == 0? trying to understand the implications of this

Reply:
we want 0 liquidity to be treated as very low liquidity, whereas not incrementing will be treated as very high liquidity